config/ecosystem-top-domain/db.ecosystem.org

```diff
@@ -50,3 +50,28 @@ LMHBM64R2VOF2IJIOGG2FXVWGYE42GF3CT7UF6EMOEWDS7ID3I2A.iso6523-actorid-upis.0088.i
 ;;iso6523-actorid-upis::0088:test02
 PUTEUP6A7HSJKCCIAOPTTGURPR5JO253IYZGAMRE2MZNUZMIA7JA.iso6523-actorid-upis.0088.iso6523.participants CNAME github-smp.publisher.0088.iso6523.participants.ecosystem.org.
 
+;; --------------------------------------
+;; Delegation as per
+;; "Option D - Federated Domain SML with partitioning by Stage and Participant Identification Scheme:
+;; (draft version 2024-11-26)
+;; with the exception that the top-level SML domain is 'participants.ecosystem.org' instead of '[prod|test].participant.peppol.org'
+;; --------------------------------------
+
+;; Some SMP entries, directly in this zone
+smp-test-01.publisher.participant NAPTR 100 10 "U" "Meta:SMP" "!.*!https://172.20.0.201/!" .
```

Comment:
Is this made intentionally, that all the SMP publisher records will be registered on  ecosystem top domain e.g. .participant.peppol.org?  Or is it better to leave that to "regional SML service to handle that"?

Reply:
This reflects the current state of the draft document (see Section 1.3.6: Publisher Zone). But indeed, I added a comment to the doc yesterday how this essentially makes the central SML a single point of failure for the second time, and that this needs to be discussed. Not sure how to handle registration in regional smls though; register in all regionals where you have participants? Or would cross-references be allowed?